test/regressions/testsBySuite.ts

```diff
@@ -24,9 +24,19 @@ const docsImports = import.meta.glob<React.ComponentType>(
     '!docsx/data/date-pickers/date-calendar/DateCalendarServerRequest', // Has random behavior (TODO: Use seeded random)
     '!docsx/data/charts/tooltip/Custom*', // Composition example
     '!docsx/data/charts/tooltip/Item*', // Composition example
+    '!docsx/data/charts/tooltip/AxisFormatter',
+    '!docsx/data/charts/tooltip/Formatting',
+    '!docsx/data/charts/tooltip/SeriesFormatter',
+    '!docsx/data/charts/tooltip/TooltipStyle',
+    'docsx/data/charts/tooltip/Interaction',
```

Comment:
Is this intended? Otherwise maybe add a comment about it?

```suggestion
    '!docsx/data/charts/tooltip/Interaction',
```